include/rogue/interfaces/ZmqClient.h

```diff
@@ -65,7 +65,7 @@ namespace rogue {
 
             std::string send(std::string value);
 
-            void close();
+            void stop();
```

Comment:
Will removing the close() break any existing builds or interfaces?

Reply:
No. ZmqClient is only really used with the VirtualClient class and close() was never actually ever called outside the destructor.